packages/reporter/src/commands/object-viewer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import _ from 'lodash'
+import Collapsible from '../collapsible/collapsible'
+
+interface ObjectViewerProps {
+  obj: Record<any, any>
+  isOpen: boolean
+}
+
+export const ObjectViewer = ({ obj, isOpen }: ObjectViewerProps) => {
+  return Object.keys(obj).length === 1 && !_.isObject(obj[Object.keys(obj)[0]])
```

Comment:
clearer definition of what this is doing.

```suggestion
  const isSingleObjValue = Object.keys(obj).length === 1 && !_.isObject(obj[Object.keys(obj)[0]])

  return Object.keys(obj).length === 1 && !_.isObject(obj[Object.keys(obj)[0]])
```